src/commands/createSwaConfigFile.ts

```diff
@@ -0,0 +1,47 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { writeFile } from 'fs-extra';
+import { join } from 'path';
+import { EntryInfo, promise } from 'readdirp';
+import { TextDocument, Uri, window, workspace } from 'vscode';
+import { ext } from 'vscode-azureappservice/out/src/extensionVariables';
+import { IActionContext } from 'vscode-azureextensionui';
+import { configFileName } from '../constants';
+import { localize } from '../utils/localize';
+import { getWorkspaceFolder } from '../utils/workspaceUtils';
+
+export async function createSwaConfigFile(context: IActionContext): Promise<void> {
+    const workspacePath: string = (await getWorkspaceFolder(context)).uri.fsPath;
+    let configFilePath: string | undefined = await getExistingConfigFilePath(workspacePath);
+
+    if (configFilePath) {
+        void ext.ui.showWarningMessage(localize('swaConfigFileExists', 'Static Web App configuration file "{0}" already exists.', configFilePath));
```

Comment:
Should probably give them the option to overwrite it if they want to start fresh.